support-frontend/assets/pages/contributions-landing/components/ContributionThankYou/utils/social.js

```diff
@@ -13,41 +17,48 @@ const LANDING_PAGE_URL_FACEBOOK = `${LANDING_PAGE_URL}?INTCMP=${INTCMP_FACEBOOK}
-    : 'Join me and over one million others in supporting a different model for open, independent journalism. Together we can help safeguard The Guardian’s future – so more people, across the world, can keep accessing factual information for free');
-
-const emailBodyCopy = (referralCode: ?string, isUsEndOfYearAppeal: ?boolean) =>
-  (isUsEndOfYearAppeal
-    ? `Join me and support the Guardian’s open, independent journalism for 2021 and beyond. You can contribute today from as little as $1. Together we’re doing something powerful. #supporttheguardian - ${emailLandingPageUrl(referralCode)}`
-    : `Join me and over one million others in supporting a different model for open, independent journalism. Together we can help safeguard The Guardian’s future – so more people, across the world, can keep accessing factual information for free: ${emailLandingPageUrl(referralCode)}`);
-
-const emailSubject = (isUsEndOfYearAppeal: ?boolean) =>
-  (isUsEndOfYearAppeal
-    ? 'Join me in supporting the Guardian'
-    : 'Join me in supporting open, independent journalism');
+const TWITTER_TEXT_COPY =
+  'Join me and over one million others in supporting a different model for open, independent journalism. Together we can help safeguard The Guardian’s future – so more people, across the world, can keep accessing factual information for free';
+const EMAIL_SUBJECT_COPY = 'Join me in supporting open, independent journalism';
+const emailBodyCopy = (referralCode: ?string) =>
+  `Join me and over one million others in supporting a different model for open, independent journalism. Together we can help safeguard The Guardian’s future – so more people, across the world, can keep accessing factual information for free: ${emailLandingPageUrl(referralCode)}`;
 
 export const getFacebookShareLink = (referralCode: ?string): string => {
-  const encodedUrl = appendReferralCode(LANDING_PAGE_URL_FACEBOOK, 'facebook', referralCode);
+  const encodedUrl = appendReferralCode(
+    LANDING_PAGE_URL_FACEBOOK,
+    'facebook',
+    referralCode,
+  );
   return `https://www.facebook.com/sharer/sharer.php?u=${encodedUrl}`;
 };
 
-export const getTwitterShareLink = (referralCode: ?string, isUsEndOfYearAppeal: ?boolean): string => {
-  const encodedUrl = appendReferralCode(LANDING_PAGE_URL_TWITTER, 'twitter', referralCode);
-  const encodedText = encodeURI(twitterTextCopy(isUsEndOfYearAppeal));
-  return `https://twitter.com/intent/tweet?url=${encodedUrl}&text=${encodedText}${isUsEndOfYearAppeal ? '&hashtags=supporttheguardian' : ''}`;
+export const getTwitterShareLink = (referralCode: ?string): string => {
+  const encodedUrl = appendReferralCode(
+    LANDING_PAGE_URL_TWITTER,
+    'twitter',
+    referralCode,
+  );
+  const encodedText = encodeURI(TWITTER_TEXT_COPY);
+  return `https://twitter.com/intent/tweet?url=${encodedUrl}&text=${encodedText}`;
```

Comment:
I wonder if we should write tests for these? It feels like it could be easy to mess these up when adding/removing things like this

Reply:
👍